src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -115,6 +122,8 @@ type MoneyRequestViewProps = {
     mergeTransactionID?: string;
 };
 
+const perDiemPoliciesSelector = (policies: OnyxCollection<OnyxTypes.Policy>) => Object.fromEntries(Object.entries(policies ?? {}).filter(([, policy]) => policy?.arePerDiemRatesEnabled));
```

Comment:
What if there are no per diems in the workspace, but the feature is enabled? 

Should we allow moving and creating reports?

Reply:
no, we shouldn't, thanks for pointing this out